packages/fxa-settings/src/pages/Index/index.tsx

```diff
@@ -96,9 +96,24 @@ export const Index = ({
     );
   }
 
+  const cmsInfo = integration.getCmsInfo();
+
   return (
     <AppLayout>
-      {isSync ? (
+      {cmsInfo ? (
+        <>
+          {cmsInfo?.shared?.logoUrl && cmsInfo?.shared?.logoAltText && (
+            <img
+              src={cmsInfo?.shared.logoUrl}
+              alt={cmsInfo?.shared.logoAltText}
+              className="justify-start mb-4 max-h-[40px]"
+            />)}
+          <h1 className="card-header">{cmsInfo?.EmailFirstPage?.headline}</h1>
+          <p className="mt-1 mb-9 text-sm">
+            {cmsInfo?.EmailFirstPage?.description}
+          </p>
```

Comment:
The optional chaining is to silence the type checker?  Or is it actually possible that there's an empty `<h1>` on the rendered content?